setup.py

```diff
@@ -92,7 +92,7 @@
     "test": ["parameterized", "pytest-cov", "pytest-rerunfailures", "pytest-xdist", "pytest"],
     # vllm is not available on Windows
     # vllm 0.7.3 causes hanging while gathering. temporary pinning the version until the issue is resolved
-    "vllm": ["vllm==0.7.2; sys_platform != 'win32'"],
+    "vllm": ["vllm==0.7.2; sys_platform != 'win32'", "uvicorn", "fastapi", "requests"],
```

Comment:
If I'm not mistaken, `uvicorn` comes with `fastapi` by [default](https://fastapi.tiangolo.com/deployment/manually/#install-the-server-program), so we don't need it here: 
```suggestion
    "vllm": ["vllm==0.7.2; sys_platform != 'win32'", "fastapi", "requests"],
```
![Screenshot 2025-03-19 at 08 46 34](https://github.com/user-attachments/assets/4b055a72-c4c7-4290-9323-97243b2797bc)